src/react/experimental/transition-manager.js

```diff
@@ -15,7 +15,7 @@ export const TRANSITION_EVENTS = {
   IGNORE: 3
 };
 
-const DEFAULT_PROPS = {
+export const DEFAULT_TRANSITION_PROPS = {
```

Comment:
Alternatively you could assign `TransitionManager.defaultProps = defaultProps` at the end.

Reply:
Agree, avoids extra import and looks more clean, updated with suggested change.